src/main/java/at/hannibal2/skyhanni/features/misc/items/EstimatedItemValueTooltip.kt

```diff
@@ -0,0 +1,24 @@
+package at.hannibal2.skyhanni.features.misc.items
+
+import at.hannibal2.skyhanni.SkyHanniMod
+import at.hannibal2.skyhanni.api.event.HandleEvent
+import at.hannibal2.skyhanni.events.item.ItemHoverEvent
+import at.hannibal2.skyhanni.skyhannimodule.SkyHanniModule
+import at.hannibal2.skyhanni.utils.ItemPriceUtils.getPrice
+import at.hannibal2.skyhanni.utils.ItemUtils.getInternalNameOrNull
+import at.hannibal2.skyhanni.utils.NumberUtil.addSeparators
+
+@SkyHanniModule
+object EstimatedItemValueTooltip {
+
+    @HandleEvent(onlyOnSkyblock = true)
+    fun onTooltip(event: ItemHoverEvent) {
+        if (!SkyHanniMod.feature.inventory.estimatedItemValues.showTooltip) return
+        val internalName = event.itemStack.getInternalNameOrNull() ?: return
+
+        val (total, _) = EstimatedItemValueCalculator.calculate(event.itemStack, mutableListOf())
+        if (total == 0.0) return
+        if (internalName.getPrice() == total) return
```

Comment:
check instead if total is base price. see https://github.com/hannibal002/SkyHanni/blob/366437a002a20dbd6ce3f698400562a3107722ef/src/main/java/at/hannibal2/skyhanni/features/misc/AuctionHousePriceComparison.kt#L65

Reply:
At this point maybe even reuse the function at https://github.com/hannibal002/SkyHanni/blob/acac160e6ea536b83e4c114ef58bc1653b91db3d/src/main/java/at/hannibal2/skyhanni/features/misc/items/EstimatedItemValueCalculator.kt#L153 and add a check that returns null if the estimated price is the same as base price